package.json

```diff
@@ -74,7 +74,7 @@
     "react-moveable": "^0.54.1",
     "react-textarea-autosize": "^8.5.3",
     "satori": "^0.10.4",
-    "sharp": "^0.32.6",
+    "sharp": "^0.32.5",
```

Comment:
Is there a reason to downgrade this or just older code? 🤔

Reply:
This was by mistake too so I fixed that

Reply:
This still hasn't been installed back as the right version. If you need a hand let me know. 🦾

Reply:
On it !